envisage/ui/tasks/tests/test_tasks_application.py

```diff
@@ -6,11 +6,14 @@
 import tempfile
 import unittest
 
-from envisage._compat import pickle
+import pkg_resources
+from six.moves import cPickle as pickle
+
 from envisage.ui.tasks.api import TasksApplication
+from envisage.ui.tasks.tasks_application import DEFAULT_STATE_FILENAME
 
 requires_gui = unittest.skipIf(
-    os.environ["ETS_TOOLKIT"] == "null",
+    os.environ.get("ETS_TOOLKIT") == "null",
```

Comment:
Drive-by fix: make this skip instead of failing if `ETS_TOOLKIT` isn't even defined.